src/build.py

```diff
@@ -886,9 +886,10 @@ def Binaryen():
 
   proc.check_call(
       [PREBUILT_CMAKE_BIN, '-G', 'Ninja', BINARYEN_SRC_DIR,
+       '-DCMAKE_BUILD_TYPE=Release',
```

Comment:
Yeah, currently Binaryen's `CMakeLists.txt` currently enables asserts for Release builds on Linux but not Windows; something I'm about to submit a PR to fix. With the asserts I'm fine making it Release everywhere.